consensus/state_processing/src/per_block_processing/process_operations.rs

```diff
@@ -76,33 +76,30 @@ pub mod base {
             )
             .map_err(|e| e.into_with_index(i))?;
 
-            match attestation {
-                AttestationRef::Base(att) => {
-                    let pending_attestation = PendingAttestation {
-                        aggregation_bits: att.aggregation_bits.clone(),
-                        data: att.data.clone(),
-                        inclusion_delay: state.slot().safe_sub(att.data.slot)?.as_u64(),
-                        proposer_index,
-                    };
-
-                    if attestation.data().target.epoch == state.current_epoch() {
-                        state
-                            .as_base_mut()?
-                            .current_epoch_attestations
-                            .push(pending_attestation)?;
-                    } else {
-                        state
-                            .as_base_mut()?
-                            .previous_epoch_attestations
-                            .push(pending_attestation)?;
-                    }
-                }
-                AttestationRef::Electra(_) => {
-                    // TODO(electra) pending attestations are only phase 0
-                    // so we should just raise a relevant error here
-                    todo!()
-                }
+            let AttestationRef::Base(attestation) = attestation else {
```

Comment:
This code path only cares about being Base or not. Using let else reduces diff :)